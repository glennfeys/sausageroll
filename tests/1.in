~ ~ ~n~ ~n~ ~ 
~ +S+s+n+s+n~ 
~ ~x~ ~s~ ~s~ 
