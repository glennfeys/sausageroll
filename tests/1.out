~ ~ ~n~ ~n~ ~ 
~ +S+s+n+s+n~ 
~ ~x~ ~s~ ~s~ 

~ ~ ~ ~ ~n~ ~ 
~ +E+x+n+s+n~ 
~ ~ ~ ~s~ ~s~ 

~ ~ ~ ~ ~ ~n~ 
~ + +E+x+n+s~ 
~ ~ ~ ~ ~s~ ~ 

~ ~ ~ ~ ~ ~ ~ 
~ + + +E+x+n~ 
~ ~ ~ ~ ~ ~s~ 

